docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -1,74 +1,150 @@
 <!--# Testing SNS Locally after Choosing Parameters-->
```

Comment:
```suggestion
# Testing SNS Locally after Choosing Parameters
```

In general: can we bring back the titles? the goal is now to release this on the website as soon as it is finished.